src/main/java/com/rarchives/ripme/App.java

```diff
@@ -39,15 +39,23 @@
     private static final History HISTORY = new History();
 
     public static void main(String[] args) throws MalformedURLException {
-        Utils.configureLogger();
-        System.setProperty("apple.laf.useScreenMenuBar", "true");
-        System.setProperty("com.apple.mrj.application.apple.menu.about.name", "RipMe");
-        logger = Logger.getLogger(App.class);
-        logger.info("Initialized ripme v" + UpdateUtils.getThisJarVersion());
-
-        if (args.length > 0) {
+        CommandLine cl = getArgs(args);
+        if (args.length > 0 && cl.hasOption('v')) {
+            System.out.println(UpdateUtils.getThisJarVersion());
+            handleArguments(args);
+        } else if (args.length > 0) {
+            Utils.configureLogger();
+            System.setProperty("apple.laf.useScreenMenuBar", "true");
+            System.setProperty("com.apple.mrj.application.apple.menu.about.name", "RipMe");
+            logger = Logger.getLogger(App.class);
+            logger.info("Initialized ripme v" + UpdateUtils.getThisJarVersion());
             handleArguments(args);
```

Comment:
I've bad luck with Eclipse like always: it's nothing straightforward to get to work, instead it always complains something the project's code refers to is missing. Therefore I can't test. I think I should use some guide to get it work right.